front/app/containers/InitiativesShow/Phone.tsx

```diff
@@ -215,14 +215,16 @@ const Phone = ({
         <ScreenReaderOnly>
           <FormattedMessage tagName="h2" {...messages.invisibleTitleContent} />
         </ScreenReaderOnly>
-        <Box mb="20px">
-          <Topics
-            postType="initiative"
-            postTopicIds={initiative.data.relationships.topics.data.map(
-              (topic) => topic.id
-            )}
-          />
-        </Box>
+        {initiative.data.relationships.topics.data.length > 0 && (
```

Comment:
Same question here - if topics.data doesn't exist, would the length check cause an error?

Reply:
If you look at FE types, the answer would be "no". But then I checked and apparently we only require topics in the FE, so the types should rather say topics is possibly undefined indeed.

So yes, I'll add an extra check. Good point.

Reply:
So currently it shouldn't happen in practice, but it _could_ happen (if we change the form requirements in the FE at one point). 

So my check is also too prudent atm. I had a proposal without topic (from the seeds) but on production platforms proposals will always have a topic atm.

Reply:
Edit: will deal with it outside of this PR